scripts/build-packages/build-packages

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+# This script creates ggshield packages in various formats.
+#
+# It expects [nfpm][] and [shiv][] to be installed.
+#
+# Packages are created in the `packages` directory.
+#
+# [nfpm]: https://github.com/goreleaser/nfpm/
+# [shiv]: https://github.com/linkedin/shiv
+
+cd $(dirname $0)
+NFPM_YAML_TMPL=$PWD/nfpm.yaml.tmpl
```

Comment:
```suggestion
NFPM_YAML_TMPL=$(dirname $0)/nfpm.yaml.tmpl
```
To remove the need to `cd`

Reply:
I do it this way to ensure the variable contains the absolute full path. That avoids potential issues.